supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -137,19 +137,19 @@ If available, inform users of alternative capabilities and workarounds.
 [discrete]
 ===== Deprecation notice
 ----
-In <product> <release>, <name of capability> is deprecated. Bug fixes and support are provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In <product> <release>, <name of capability> is deprecated and will be removed in the <deprecation time frame>. Bug fixes and support are provided through the end of the current release life cycle, but be aware that deprecated items receive no further feature enhancements and will be removed.
```

Comment:
```suggestion
In <product> <release>, <name of capability> is deprecated and is planned to be removed in the <deprecation time frame>. Bug fixes and support are provided through the end of the current release life cycle. Deprecated items do not receive feature enhancements.
```
Does this suggestion align with the following example and improve clarity?

Reply:
If adopted, apply changes to the following example, too.

Reply:
`<deprecation time frame>` seems ambiguous to me. In what units is this time frame counted? If it's defined in terms of a future release, then it'd be very helpful to me read just that inside `<...>` so that I don't have to stop and think and ask or do something different because I'm pressed for time and can't think straight.

Reply:
Same as for the removal note:

A while back, I had a conversation with an SME who asked me to insert the Optional sentence as the second sentence:

```suggestion
In <product> <release>, <name of capability> is deprecated and will be removed in the <deprecation time frame>. (Optional: You can use <alternative capability> instead.) Bug fixes and support are provided through the end of the current release life cycle, but be aware that deprecated items receive no further feature enhancements and will be removed.
```
![image](https://user-images.githubusercontent.com/78564304/155340740-54e91ec3-b6aa-4aa4-8d45-1410fdefa2e0.png)

Reply:
I also added underscores to the user-replaced values in my suggestion according to https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

Reply:
> I also added italics and underscores to the user-replaced values in my suggestion according to https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

The other samples in the SSG don't use italics, so to remain consistent I'm not adding them here